packages/manager/modules/sign-up/src/components/siret/siret.service.js

```diff
@@ -0,0 +1,23 @@
+export default class SiretService {
+  /* @ngInject */
+  constructor($http, $locale) {
+    this.$http = $http;
+    this.$locale = $locale;
+  }
+
+  getSiret(params) {
+    return this.$http
+      .get('/me/suggest/company', { params })
+      .then(({ data }) => data)
+      .catch(() => ({
+        entryList: [],
+      }));
```

Comment:
**question**: doing so prevents any errors from being catched on this promise. Is this what we want ?

Reply:
Currently the errors are not really interpretable and could not be translated.
The best thing to do is not to have any results and to propose to the user to switch to manual mode.